app/assets/javascripts/application.js

```diff
@@ -85,6 +85,8 @@ $(function() {
   $('a[href*=#]:not([href=#])').click(function() {
     if (location.pathname.replace(/^\//,'') == this.pathname.replace(/^\//,'') && location.hostname == this.hostname) {
       var target = $(this.hash);
+      preventDefault()
```

Comment:
Oh I meant removing the entire line.

In a normal situation, you would have a function like this:

html
```
<div onclick="thefunction()"></div>
```

js:
```
var thefunction = function(e){ // <-- notice the e, this is where it gets initialized, it's the event.
  e.preventDefault() // preventDefault gets called on e, it's a function defined on the event.
  // do stuff
}